core/templates/dev/head/pages/exploration_player/PlayerTranscriptService.js

```diff
@@ -76,6 +77,13 @@ oppia.factory('PlayerTranscriptService', ['$log', function($log) {
       });
       numAnswersSubmitted = 0;
     },
+    addPreviousCard: function() {
+      if (transcript.length === 1) {
+        return;
+      }
+      var previousCard = transcript[transcript.length - 2];
```

Comment:
You might want to add a TODO here to generalize later to the case where the "concept card" actually takes up multiple cards (depending on how worked examples are introduced).

Reply:
Done.